src/scripts/main.js

```diff
@@ -354,7 +354,25 @@ const people = [
   },
 ];
 
-// eslint-disable-next-line no-console
-console.log(people); // you can remove it
+const dashboard = document.querySelector('.dashboard');
 
-// write your code here
+people.forEach(person => {
+  const row = document.createElement('tr');
+
+  const cellsData = [
+    person.name,
+    person.sex === 'm' ? 'Мужской' : 'Женский',
+    person.born,
+    person.died,
+    person.died - person.born,
+    Math.ceil(person.died / 100)
```

Comment:
The calculation for the century using `Math.ceil(person.died / 100)` might not be accurate for years ending in '00'. For example, the year 1900 should be in the 19th century, but this calculation would place it in the 20th century. Consider using `Math.ceil((person.died - 1) / 100)` to handle such cases correctly.